### DISCUSSION TITLE
Use with CocoaPods plugin

### DISCUSSION THREAD

**Discussion by joreilly** [touchlab/Kermit#68] [Q&A]
All the examples I've seen seem to make use of "co.touchlab.native.cocoapods" ....is there example of exporting using "standard" CocoaPods plugin (https://kotlinlang.org/docs/reference/native/cocoapods.html)?

**Comment by samhill303** [comment]
The stock CocoaPods plugin doesn't allow for configuring the framework to export kermit and since the plugin configures the compilation you can't export when configuring the target like you might otherwise. I wasn't able to find a way to update the binary at a later point when using the plugin. Of course, if you don't need to use Kermit from native iOS then you don't need to export and the stock plugin should work. Or if you don't need to use cocoapods you can export to the target binary as shown here https://kotlinlang.org/docs/reference/mpp-build-native-binaries.html#export-dependencies-to-binaries

**Comment by joreilly** [comment]
Thanks @samhill303, will take a look at that
